spec/models/actor_movie_spec.rb

```diff
@@ -0,0 +1,4 @@
+class ActorMovie < ApplicationRecord
+  belongs_to :actor
```

Comment:
I think you might've copied the model here but this spec file shoud include assertions like this:
`    it {should belong_to :movie}`
`    it {should belong_to :actor}`